microprofile-health/pom.xml

```diff
@@ -45,8 +39,11 @@
       <artifactId>rest-assured</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>io.quarkus</groupId>
+      <artifactId>quarkus-smallrye-health</artifactId>
```

Comment:
Let's put that one at the top.

(I suppose you have generated the file again but it's more readable for the user this way so I think it's worth handcrafting a few things)